spec/support/vcr.rb

```diff
@@ -10,8 +10,4 @@
   end
 
   config.filter_sensitive_data('<API_KEY>') { ENV['NEWGISTICS_API_KEY'] }
-
-  config.default_cassette_options = {
```

Comment:
does this mean that VCR will record by default now if we don't have a cassette for a request?

Reply:
Yup, if you don't have a cassette it records a new one, if you have one and the requests made on the test don't match the existing cassette it throws an error.